package.json

```diff
@@ -114,5 +116,8 @@
     "ie 8",
     "ie 9",
     "iOS 9"
-  ]
+  ],
+  "overrides": {
+    "playwright": "$playwright-core"
```

Comment:
otherwise sometimes it can try to use a playwright browser we didn't download on install